lib/modules/datasource/terraform-module/index.ts

```diff
@@ -19,7 +19,10 @@ export class TerraformModuleDatasource extends TerraformDatasource {
     super(TerraformModuleDatasource.id);
   }
 
-  override readonly defaultRegistryUrls = ['https://registry.terraform.io'];
+  override readonly defaultRegistryUrls = [
+    'https://registry.terraform.io',
+    'https://app.terraform.io',
```

Comment:
This change will mean that Renovate attempts to look up every single package from both registries. Are you sure that's correct?

Reply:
Do not add this to the defaultRegistryUrls. Renovate will attempt to retrieve every module it can not fund publicly at `app.terraform.io`. 

Rather than doing that, create a new readonly attribute which holds Urls that support the extended API.